code/game/movement.dm

```diff
@@ -0,0 +1,117 @@
+//Movement could literally be its own module
```

Comment:
I mean, ya, it COULD be its own module, but I really loved how the first proc in movable.dm was Move(). There was just something serene about that level of unity